apps/admin-x-settings/src/components/settings/membership/member-emails/welcome-email-modal.tsx

```diff
@@ -168,58 +172,38 @@ const WelcomeEmailModal = NiceModal.create<WelcomeEmailModalProps>(({emailType =
                                 )}
                             </div>
                             <Button
-                                color="black"
-                                disabled={!hasChanges || isSaving}
-                                label={isSaving ? 'Saving...' : 'Save'}
-                                onClick={handleSave}
+                                color={okProps.color}
+                                disabled={okProps.disabled ?? saveState === 'saving'}
```

Comment:
```suggestion
                                disabled={okProps.disabled'}
```

The additional `saveState === 'saving'` is redundant; see [useForm](https://github.com/TryGhost/Ghost/blob/770f657ae91e95a9874be0909a08db315c205962/apps/admin-x-framework/src/hooks/useForm.ts#L143)

Reply:
oh good call!